### ISSUE TITLE
Medieval Dynasty (1129580)

### ISSUE THREAD

**Issue by wofferl** [ValveSoftware/Proton#4219]
# Compatibility Report
- Name of the game with compatibility issues: Medieval Dynasty 
- Steam AppID of the game: 1129580

## System Information
- GPU: Nvidia RTX 2080 TI
- Driver/LLVM version: Nvidia 455.23.04 
- Kernel version: 5.8.10-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/wofferl/93a321042c8d43422520619ac444354c):
- Proton version: Proton-5.9-GE-6-ST, also tested other versions

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1129580.log](https://github.com/ValveSoftware/Proton/files/5282308/steam-1129580.log)

## Symptoms 
Cut scenes only work with the Proton-GE build.
No Audio at all.
Installed Faudio, but didn't help.

> protontricks 1129580 faudio
> protontricks 1129580 xact
> 

```
14661.341:00e4:00e8:trace:loaddll:load_native_dll Loaded L"C:\\windows\\system32\\FAudio.dll" at 0x6d840000: native
14661.341:00e4:00e8:trace:loaddll:load_native_dll Loaded L"C:\\windows\\system32\\X3DAudio1_7.dll" at 0x668c0000: native
14664.329:00e4:00e8:trace:loaddll:load_native_dll Loaded L"Z:\\data\\steam\\steamapps\\common\\Medieval Dynasty\\Engine\\Binaries\\ThirdParty\\Windows\\XAudio2_9\\x64\\xaudio2_9redist.dll" at 0x33d0000: native
14664.378:00e4:00e8:trace:loaddll:load_native_dll Loaded L"C:\\windows\\system32\\XAudio2_7.dll" at 0x15eb0000: native
```

## Reproduction

Start the game.

**Comment by kisak-valve** [comment]
Hello @wofferl, does running the game with Proton 4.11, then with Proton 5.0 have an effect?

**Comment by wofferl** [comment]
> Hello @wofferl, does running the game with Proton 4.11, then with Proton 5.0 have an effect?

Thanks,

I tested this with a fresh prefix and audio works with 4.11, when xact and faudio is NOT installed. When then switching to 5.0 the audio doesn't work again.

This is the log with working audio and proton 4.11:
[steam-1129580.log](https://github.com/ValveSoftware/Proton/files/5282696/steam-1129580.log)

I also forgot, that with original proton, the cut scenes are black. They work with the GE proton build.

**Comment by wofferl** [comment]
Sorry, it also works with 5.0 then. Unfortunately the game has set the main volume to 0 when switching to 5.0 in the first try.

It now also works with the Proton-5.9-GE-6-ST build.

It seems that the XAudio2_7.dll is not loading now and only the audio2_9redist.dll is used.

Here the log with the working Proton-5.9-GE-6-ST build.
[steam-1129580.log](https://github.com/ValveSoftware/Proton/files/5282803/steam-1129580.log)

**Comment by kisak-valve** [comment]
So, what I suspect is happening here is that there's a situation where XAudio isn't playing nicely with win10 wine prefixes (most notably with Halo MCC). Proton 4.11 forces a win7 wine prefix while Proton 5.0 leaves existing prefixes alone, but also sets the wine prefix to win10 on fresh setups.

This isn't technically a Proton regression because the game is choosing to exercise a different audio path based on what OS it thinks it's running on. (It's still an issue, but something that was hidden in older Proton builds.)

**Comment by wofferl** [comment]
You are right, I tried with a fresh 5.0 prefix and set the it to win7 (protontricks 1129580 win7) and it also works.

**Comment by Fantu** [comment]
Before install the first time the game I apply this workaround found on protondb:
- Download https://aka.ms/vs/16/release/vc_redist.x64.exe
- Copy into .steam/steam/steamapps/common/Medieval Dynasty/Engine/Extras/Redist/en-us/
- Rename “UE4PrereqSetup_x64.exe → UE4PrereqSetup_x64.exe.old“ and “VC_redist.x64.exe → UE4PrereqSetup_x64.exe“

With proton version 6.3-7 always freeze on after second logo
With Proton-GE version 6.21-GE-1 works but have occasional input lag (I not found exactly what trigger it), someone have a method or workaround to solves also the input lag please?

I have/use:
Ubuntu 20.04.3
CPU: AMD Ryzen 5 5600x
GPU: Nvidia GTX 1650 with Nvidia 470 (from ubuntu repository)

**Comment by kleener** [comment]
Hi,
my only problem at the moment is that the game starts only once after installation or after an update. It remains after the logos the screen black. The problem is also only on the Nvidia GTX 1070, on my laptop with an internal AMD card it works without problems to start several times.
Do you have any clues?

I use:
Kubuntu 21.10
GPU: Nvidia GTX 1070 with Nvidia 495

**Comment by kleener** [comment]
So I have no more problems, it runs stable and without problems. I guess you can close the ticket.

**Comment by kisak-valve** [comment]
Medieval Dynasty (1129580)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9300.
@Markvis posted on 2025-12-11T18:05:20:

# Compatibility Report
- Name of the game with compatibility issues: Medieval Dynasty
- Steam AppID of the game: 1129580

## System Information
- GPU: RTX 3080
- Video driver version: nvidia 580.105.08
- Kernel version: 6.17.10-300.fc43.x86_64
- Link to full system information: [Gist](https://gist.github.com/Markvis/b868512924af16d001451134519a9aea)
- Proton version: Proton 10.0-3

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1129580.tar.gz](https://github.com/user-attachments/files/24109256/steam-1129580.tar.gz)

## Symptoms 

Creation Wheel Stuck on Proton 10.0-3 but works fine on 9.0-4

<img width="1056" height="994" alt="Image" src="https://github.com/user-attachments/assets/6ac3ac2a-907e-4361-923b-3b4cbf202539" />


## Reproduction

1. Launch game with Proton version 10.0-3
2. Start new game or load a save file
3. Press Q button to open the creation wheel / building wheel
4. Using mouse cursor try to select other sections of the wheel
5. The wheel cursor is stuck on the upper left

**Comment by Markvis** [comment]
looks like above issue was related to 
https://github.com/ValveSoftware/Proton/issues/8681

Controller also didnt work but when I set `PROTON_PREFER_SDL=1` the issues went away. the controller and the stuck creation wheel when using mouse and keyboard
